pkg/tsdb/cloudmonitoring/cloudmonitoring_test.go

```diff
@@ -997,6 +1061,16 @@ func loadTestFile(path string) (cloudMonitoringResponse, error) {
 	return data, err
 }
 
+func getQueriesFromInterface(qis []cloudMonitoringQueryInterface) []*cloudMonitoringQuery {
```

Comment:
Since it returns an array of `*cloudMonitoringQuery` I would suggest renaming it:
```suggestion
func getCloudMonitoringQueriesFromInterface(qis []cloudMonitoringQueryInterface) []*cloudMonitoringQuery {
```